web/src/beta/features/Editor/Map/LayersPanel/LayerItem.tsx

```diff
@@ -42,11 +44,11 @@ const LayerItem: FC<LayerItemProps> = ({
     handleLayerDelete,
     handleLayerNameUpdate,
     handleLayerVisibilityUpdate,
-    handleFlyTo,
-    openCustomPropertySchema,
-    handleCustomPropertySchemaClick
+    handleFlyTo
   } = useMapPage();
 
+  const [isVisible, setIsVisible] = useState(false);
```

Comment:
Could you naming it more specific? It's `LayerItem.tsx`, naming it like this could mislead other devs thought it's the visiblity of layer item.